### DISCUSSION TITLE
How to specify the path for HDFS cold storage

### DISCUSSION THREAD

**Discussion by fdf1779** [apache/doris#35834] [A - General / Q&A]
CREATE RESOURCE "remote_hdfs" PROPERTIES (
        "type"="hdfs",
        "fs.defaultFS"="fs_host:default_fs_port",
        "hadoop.username"="hive",
        "hadoop.password"="hive",
        "dfs.nameservices" = "my_ha",
        "dfs.ha.namenodes.my_ha" = "my_namenode1, my_namenode2",
        "dfs.namenode.rpc-address.my_ha.my_namenode1" = "nn1_host:rpc_port",
        "dfs.namenode.rpc-address.my_ha.my_namenode2" = "nn2_host:rpc_port",
        "dfs.client.failover.proxy.provider" = "org.apache.hadoop.hdfs.server.namenode.ha.ConfiguredFailoverProxyProvider"
    );
When I use HDFS as remote storage, I am unable to specify a storage path, and Doris will default to storing it under/data

**Comment by ixzc** [comment]
it will store the data in the default path,
we have change this : https://github.com/apache/doris/pull/32632
 and will merge this to 2.1 : https://github.com/apache/doris/pull/35848
